apps/www/registry/new-york/ui/command.tsx

```diff
@@ -29,6 +31,10 @@ const CommandDialog = ({ children, ...props }: CommandDialogProps) => {
   return (
     <Dialog {...props}>
       <DialogContent className="overflow-hidden p-0">
+        <VisuallyHidden>
+          <DialogTitle>Title</DialogTitle>
+          <DialogDescription>Description</DialogDescription>
+        </VisuallyHidden>
```

Comment:
AFAIK, `DialogDescription` is optional. 
And is there a reason to use `<VisuallyHidden />` instead of `className="sr-only"`?

```suggestion
        <DialogTitle className="sr-only">Title</DialogTitle>
```